templates/patroni.yml.j2

```diff
@@ -0,0 +1,27 @@
+bootstrap:
+  dcs:
+    postgresql:
+      use_pg_rewind: true
+  initdb:
+  - auth-host: md5
+  - auth-local: trust
+  - encoding: UTF8
+  - locale: en_US.UTF-8
+  - data-checksums
+  pg_hba:
+  - host all all 0.0.0.0/0 md5
+  - host replication replication {{ pod_ip }}/16 md5
+bypass_api_service: true
+log:
+  dir: /var/log/postgresql
+restapi:
+  connect_address: '{{ pod_ip }}:8008'
+  listen: 0.0.0.0:8008
+pod_ip: '{{ pod_ip }}'
+postgresql:
+  connect_address: '{{ pod_ip }}:5432'
+  custom_conf: /var/lib/postgresql/data/postgresql-k8s-operator.conf
+  data_dir: /var/lib/postgresql/data/pgdata
```

Comment:
@marceloneppel is the data_dir equivalent to the storage_path?

Reply:
`data_dir` is a folder inside `storage_path`. Your comment made me think about maybe I can use the STORAGE_PATH parameter here to not duplicate the path again. Thanks for the comment.